lib/innerRel.ml

```diff
@@ -536,9 +538,7 @@ and module Elts = MySet.Make(O) =
 
     let _all_topos1 nodes edges =
       do_all_topos nodes
-        (filter
-           (fun (e1,e2) -> Elts.mem e1 nodes && Elts.mem e2 nodes)
-           edges)
+        (filter_elts (fun e -> Elts.mem e nodes) edges)
```

Comment:
_all_topos1 (and do_all_topos above) is dead code. Perhaps it is time to delete?